string name
string model_path
float32 x_pos
float32 y_pos
float32 z_pos
float32 roll
float32 pitch
float32 yaw
float32 x_scale
float32 y_scale
float32 z_scale
---
bool success